test/ha/activator_test.go

```diff
@@ -52,6 +53,11 @@ func TestActivatorHA(t *testing.T) {
 		t.Fatalf("Deployment %s not scaled to %d: %v", activatorDeploymentName, haReplicas, err)
 	}
 
+	// Wait for Controller to elect a new leader to prevent race conditions.
+	if _, err := getLeader(t, clients, controllerDeploymentName); err != nil {
```

Comment:
Is it not good idea to make sure that `lease` in knative-serving ns is created in `test/e2e-tests.sh` rather than wait it here?

Reply:
Yeah. I had the Go function ready so I used it but it can be also done from Bash. Let me check how nasty that command will be.

Reply:
Done in a new commit.